.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+#*********************************************************************
+# Copyright (c) Intel Corporation 2023
+# SPDX-License-Identifier: Apache-2.0
+#*********************************************************************/
+
+# This workflow will release new versions when required using semantic-release
+
+name: Semantic-Release CI
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Use Node.js 18.x
+      uses: actions/setup-node@v3
+      with:
+        node-version: "18.x"
+    - run: npm ci
+    - run: npm run build-ext --if-present
+    - run: rsync -a package.json README.md ./dist/
+    - name: Semantic Release
+      uses: cycjimmy/semantic-release-action@v3
+      with:
+        semantic_version: 19.0.5  # It is recommended to specify a version range
```

Comment:
is this the latest version?

Reply:
Yes, as per the link https://github.com/cycjimmy/semantic-release-action I updated it.